readthedocsext/theme/templates/builds/partials/version_list.html

```diff
@@ -17,10 +17,26 @@
 
 {% block create_button %}
   {% if is_project_admin %}
-    <a href="{% url "project_version_create" project.slug %}" class="ui primary button">
-      <i class="fa-solid fa-plus icon"></i>
-      <span class="text">{% trans "Add version" %}</span>
-    </a>
+    <div class="right aligned eight wide computer eight wide tablet sixteen wide left aligned mobile middle aligned column">
+      {% blocktrans trimmed asvar text_resync %}
+        Resync versions
+      {% endblocktrans %}
+      <readthedocs-menu-sync-all-versions
+                                 class="ui primary button"
```

Comment:
I would make it more discrete here, since it's not the primary action of this page.

```suggestion
                                 class="ui button"
```